_data/mentors.yml

```
@@ -1026,4 +1026,33 @@
     extra: |
       Software Development Strategies, Resume Review, Preparation for The Technical Interview
   network:
-    - linkedin: https://www.linkedin.com/in/liliia-rafikova-635328104/
\ No newline at end of file
+    - linkedin: https://www.linkedin.com/in/liliia-rafikova-635328104/
+
+- name: Monique Grinstein
+  disabled: false
+  sort: 200
```

```suggestion
  sort: 100
```

This mentor has only 2 hours availability, we are sorting on top mentors that are available in the next months and have more ours.

---

Thanks! Didn’t know what this was for exactly :)